app/mailers/instructor_notification_mailer.rb

```diff
@@ -15,6 +15,10 @@ def email(alert)
     unless @alert.sender_email.nil?
       params[:bcc] = @alert.sender_email # sender_email gives user email of the sender
     end
+    if bcc_to_salesforce && !@alert.sender_email.nil?
+      params[:bcc] ||= []
+      params[:bcc] << ENV['SALESFORCE_BCC_EMAIL']
```

Comment:
I suggest mirroring the implementation in ticket_notification_mailer.

Reply:
I have made the code changes, can you please have a look at it?